### PR TITLE
feat(docker): add docker node support

### PR DESCRIPTION
**Pull request by LuttyYang**
According to the #153 requirements, we need to support starting the RPC node with Docker. This PR implements the functionality to start a Story's RPC node and localnet using Docker.

issue: #153

### CODE REVIEW

File: docker/localnet/script/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:22.04
```
Comment (LeoHChen) [inline docker/localnet/script/Dockerfile:1]: can you use a lightweight base image?

Comment (LuttyYang) [reply docker/localnet/script/Dockerfile:1]: Coinbase requires using Ubuntu 20 or higher as the base image, rather than Alpine or Debian.

**Comment by jdubpark** [comment]
Is there a way for us to unify modifications to the genesis and config files in `node-launcher` with the genesis and config files in 1+1, 1+2, and 1+4? A lot of moving pieces otherwise.

**Comment by edisonz0718** [comment]
The localnet is set up in another repo

**Pull request closed without merging**